docs/ir/ir-breaking-changes.rst

```diff
@@ -33,3 +33,37 @@ Consequently, if the padding space within a struct is used to store data (e.g. i
     }
 
 We have the same behavior for implicit delete, for example when array of structs is shortened.
+
+ * In case of inheritance, order of initialization has changed.
+
+Before order was:
```

Comment:
```suggestion
The order used to be:
```

Reply:
Maybe we also need to mention that before the 3 items, all state variables are 0?

Reply:
I will add that as well. I was just thinking that it shold derive from fact that storage is always zeroed out :) But, better to mention anyway